Tutorial.md

```diff
@@ -67,76 +72,54 @@ In this workflow, variants are prioritized, annotated, and filtered to aid in th
 | [svpack](https://github.com/PacificBiosciences/svpack) | annotate structural variant calls with functional consequence based on position and reference genome annotation |
 | [slivar](https://github.com/brentp/slivar) | convert filtered and annotated variants from slivar and svpack into TSVs for easy browsing |
 | [yak](https://github.com/lh3/yak) | create k-mer database for each parent (only if trios included in cohort) |
-| [hifiasm](https://github.com/chhylp123/hifiasm)                 | assemble reads with trio binning (only if trios included in cohort) | 
+| [hifiasm](https://github.com/chhylp123/hifiasm)                 | assemble reads with trio binning (only if trios included in cohort) |
 | [calN50](https://github.com/lh3/calN50)                      | calculate assembly stats  (only if trios included in cohort) |
 | [seqtk](https://github.com/lh3/seqtk) | split assembly contigs into 200kb chunks to facilitate visualization of aligned assembly with IGV (only if trios included in cohort) |
 | [minimap2](https://github.com/lh3/minimap2)                     | align assembly to reference (only if trios included in cohort) |
 
 [Back to top](#TOP)
 
----------------
+---
+
+## Getting Started
 
-# Getting Started
 These workflows are designed to be implemented on a **linux cluster** and require that you follow the installation and configuration procedures described below.
 
-# 1. Dependencies
+### 1. Dependencies
+
 - [singularity>=3.5.3](#singularity) installed by root
+  - [Singularity](https://sylabs.io/guides/3.6/admin-guide/installation.html#installation-on-linux) **must be installed with root privileges**. If you do not have root privileges and singularity is not installed on your linux cluster, we recommend contacting your HPC administrator for assistance.
 - [conda](#conda)
```

Comment:
Modify environment.yaml to match what we know about current issues with snakemake.

```yaml
- python=3
- snakemake=6
- lockfile=0.12
- mamba
```

Reply:
I put 6.15 in the environment.yaml because that's what I tested. How about snakemake=6.5.2 to include [the most recent updates to tmpdir](https://snakemake.readthedocs.io/en/stable/project_info/history.html#id98). Would that be compatible with other known snakemake issues?